From 04c29ebc3af2db34d716984a9060e1845d2679ec Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Thu, 30 Oct 2008 14:37:56 +1100
Subject: [PATCH] xserver-kdrive-common: provide xserver ppm splash support for palmt650

if a /usr/share/pixmaps/xsplash-qvga-square.ppm file exists (provided by any
optional splash screen package) display this splash on the palmt650 when x
starts.
---
 .../xserver-kdrive-common/openmoko/Xserver         |    5 +++++
 .../xserver-kdrive-common_0.1.bb                   |    2 +-
 2 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/packages/xserver-kdrive-common/xserver-kdrive-common/openmoko/Xserver b/packages/xserver-kdrive-common/xserver-kdrive-common/openmoko/Xserver
index 0965d6e..07d1192 100644
--- a/packages/xserver-kdrive-common/xserver-kdrive-common/openmoko/Xserver
+++ b/packages/xserver-kdrive-common/xserver-kdrive-common/openmoko/Xserver
@@ -98,6 +98,11 @@ case `module_id` in
 		  PPM="-root-ppm /usr/share/pixmaps/xsplash-vga.ppm"
 		fi
                 ARGS="$ARGS -dpi 285 -screen 480x640 -hide-cursor $PPM vt1" ;;
+        "Palm Treo 650")
+	        if [ -f "/usr/share/pixmaps/xsplash-qvga-square.ppm" ]; then
+		  PPM="-root-ppm /usr/share/pixmaps/xsplash-qvga-square.ppm"
+		fi
+                ARGS="$ARGS -dpi 181 -screen 320x320 -hide-cursor $PPM" ;;
         "Motorola Ezx Platform")
 	        if [ -f "/usr/share/pixmaps/xsplash-qvga.ppm" ]; then
 		  PPM="-root-ppm /usr/share/pixmaps/xsplash-qvga.ppm"
diff --git a/packages/xserver-kdrive-common/xserver-kdrive-common_0.1.bb b/packages/xserver-kdrive-common/xserver-kdrive-common_0.1.bb
index a1826e5..a49d4fa 100644
--- a/packages/xserver-kdrive-common/xserver-kdrive-common_0.1.bb
+++ b/packages/xserver-kdrive-common/xserver-kdrive-common_0.1.bb
@@ -2,7 +2,7 @@ DESCRIPTION = "Common X11 scripts"
 LICENSE = "GPL"
 SECTION = "x11"
 RDEPENDS_${PN} = "xmodmap libxrandr xdpyinfo xtscal xinit"
-PR = "r31"
+PR = "r34"
 
 SRC_URI = "\
   file://Xdefaults \
-- 
1.5.4.3

